uwblueprint/focus-on-nature
# feat: camp overview - consolidating campers list and camp details, statuses

## Notion ticket link

[Camp Statuses](https://www.notion.so/uwblueprintexecs/Task-Board-S22-1a3e651ab8544192bf0fdcbba04f6d03?p=6304568949a048ebaf6bc3b1ea2ef461&pm=s)


## Implementation description
frontend: 
* putting camp details and campers lists on the same page
* downscaling fonts 
* updating styling on camp overview page to match designs better
* updating camp status type

backend:
* returning campsessions and its campers/waitlist on get camp requests
  * we have a separate endpoint for getting the campers/waitlist for a campsession individually that we were initially going to use, BUT we don't want to use this since this requires us to make a request for each campsession
* stop passing formquestions in camp creation endpoint
  * we create formquestions in a separate endpoint

![image](https://user-images.githubusercontent.com/40839109/188284087-2f98b37e-f7b6-40d2-9c9e-1b45f704cf65.png)


## Steps to test
1. Navigate to camp detail pages to verify that the all details are correct

Some sample urls:
Published: http://localhost:3000/admin/camp/63139c7bc3d7b55b44a01531
Completed: http://localhost:3000/admin/camp/630a63450eb59852f9d01b1e
Draft: http://localhost:3000/admin/camp/630ae35648363c600c53b42e


## What should reviewers focus on?
* 


## Checklist
